src/styles/abstract/_cssVariables.scss

```diff
@@ -45,9 +45,9 @@
   --mention-color-rgb: 30, 50, 92;
   --mention-highlight-bg-rgb: 255, 212, 112;
   --mention-highlight-link-rgb: 27, 29, 34;
-  --mention-highlight-bg-mixed-rgb: 255,233,183;
-  --pinned-highlight-bg-mixed-rgb: 255,244,220;
-  --own-highlight-bg-rgb: 245,204,110;
+  --mention-highlight-bg-mixed-rgb: 255, 233, 183;
+  --pinned-highlight-bg-mixed-rgb: 255, 244, 220;
+  --own-highlight-bg-rgb: 245, 204, 110;
```

Comment:
we should not copy variables from MM to our local because in the future it may be possible that MM changes the value of these but they are not applying to our components because in our local it's not updated

but I think here the idea here is to use these values as a fallback and on changing the theme the MM's values are getting used instead of locals.